modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,6 +1,22 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+// TODO don't know release date yet
+== Neo4j 2025.XX
```

Comment:
More a comment for @renetapopova I do not think we have added all new GQLSTATUS codes here, at least for errors. The ones for notifications we have added though. Do we need some better guidelines around this?

Reply:
My understanding from the discussion yesterday is that we are removing the changelog anyway, so I have removed the changes to this file.